### PR TITLE
Chore/add php cs fixer

### PR DESCRIPTION
**Pull request by sunchayn**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,12 +12,14 @@
...
+        "php-cs-fix": "./vendor/bin/php-cs-fixer fix -v --config .php-cs-fixer.dist.php src/ tests/ publishable/"
     },
     "require": {
-      "php": "^8.1",
-      "illuminate/support": "^9",
-      "illuminate/http": "^9"
+        "php": "^8.1",
+        "illuminate/support": "^9",
+        "illuminate/http": "^9",
+        "friendsofphp/php-cs-fixer": "^3.9"
```
Comment (juampi92) [inline composer.json]: require dev

File: composer.json [hunk 2]
```diff
@@ -12,12 +12,14 @@
     ],
     "scripts": {
         "test": "phpunit",
-        "phpstan": "phpstan analyse --level=7 src"
+        "phpstan": "phpstan analyse --level=7 src",
```
Comment (juampi92) [inline composer.json:15]: can you draft a quick config? that way we can have a baseline easily and use larastan too

Comment (juampi92) [reply composer.json:15]: Ah, there is a config already `phpstan.neon.dist`. Can you import larastan there?

File: composer.json (hunk 1 above)
Comment (sunchayn) [reply composer.json]: 59fec15e082540aa7a3e984b6675e449f5be22db

File: composer.json (hunk 2 above)
Comment (sunchayn) [reply composer.json:15]: 28c1fd4ed6ef9d5d9ef71adcc66b8c03b30b20ab

I had to remove some Laravel-related parameters

Screenshot 2022-08-03 at 14 18 05

**Review by juampi92 — APPROVED** [review]
mergeee

**Pull request merged**